sourmash/sbtmh.py

```diff
@@ -14,10 +16,16 @@ def load_sbt_index(filename, print_version_warning=True):
                     print_version_warning=print_version_warning)
 
 
-def create_sbt_index(bloom_filter_size=1e5, n_children=2):
+def create_sbt_index(bloom_filter_size=1e5, n_children=2, not_localized=False):
     "Create an empty SBT index."
     factory = GraphFactory(1, bloom_filter_size, 4)
-    tree = SBT(factory, d=n_children)
+    if not_localized or n_children != 2:
+        if n_children != 2:
+            notify("n_children != 2 in tree, defaulting to non-localized sequence "
```

Comment:
I think this should raise a `ValueError` instead, since this is an unlikely situation for anyone using the command line to run into!